search.py

```diff
@@ -0,0 +1,66 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+ 
+
+#This is used to load the .env file
+import sys
+import dotenv
+import louis.models.openai as openai
+import louis.db as db
+
+dotenv.load_dotenv()
+
+
+def search(cursor, query_embedding):
+    """Match documents with a given query."""
+    data = {
+        'text': "text",
+        'query_embedding': query_embedding,
+        'match_threshold': 0.5,
+        'match_count': 10,
+        'weights': {
+            "1",
```

Comment:
this is definitely not right, where are the keys? and the value is a float between 0 and 1, not a string.